src/actions/transactions.test.js

```diff
@@ -10,82 +8,65 @@ import Fees from '../constants/fees';
 import networks from '../constants/networks';
 import { toRawLsk } from '../utils/lsk';
 
+jest.mock('../utils/api/transactions');
+jest.mock('../utils/api/delegate');
```

Comment:
I really like the jest way of mocking 👍 It saves us from a bunch of boilerplate code further down in the file.